yarn-project/sequencer-client/src/sequencer/tx_validator.ts

```diff
@@ -186,4 +188,55 @@ export class TxValidator {
 
     return VALID_TX;
   }
+
+  async #validateFee(tx: Tx): Promise<TxValidationStatus> {
+    if (!tx.data.needsTeardown) {
```

Comment:
Is this accidentally duplicated from above?

Reply:
Separately, if the sequencer is configured to expect a fee, shouldn't there always be a teardown? Or does that feel a little heavy-handed?

Reply:
I was thinking we could add a `SEQ_CHARGE_FEES` env var that would control whether the sequencer rejects txs without teardown/fee payment info. We could toggle this in tests through the `node.setConfig({ chargeFees: true })` after setup is complete. WDYT?